desktop/core/src/desktop/js/api/utils.ts

```diff
@@ -53,6 +54,36 @@ export interface ApiFetchOptions<T, E = string> extends AxiosRequestConfig {
   ) => void;
 }
 
+const axiosInstance = axios.create();
+
+let baseUrl = (window as hueWindow).HUE_BASE_URL;
+let bearerToken: string | undefined;
+
+axiosInstance.interceptors.request.use(config => {
+  if (baseUrl) {
+    config.baseURL = baseUrl;
+  }
+  if (bearerToken) {
+    config.headers['Authorization'] = `Bearer ${bearerToken}`;
```

Comment:
Bearer --> JWT ? (tried with compose API? only supported there AFAIK)

I will update the UI there so use the compo, will be easy to test!

Reply:
Bearer is the standard, https://jwt.io/introduction <- search for "bearer"

Reply:
Yes maybe, but won't work with our libs, JWT is what we use and not wrong in itself

Reply:
Probably for old reasons, you can configure with JWT_AUTH_HEADER_PREFIX': 'Bearer'

Reply:
Indeed, perfecto, gracias!